Slim/Player/Squeezebox.pm

```diff
@@ -72,12 +72,18 @@ sub reconnect {
 
 	if (!$reconnect) {
 
+		# when reconnecting after player has been forgotten, we need to power-cycle it
+		if (!defined $reconnect && $prefs->client($client)->get('power')) {
```

Comment:
Isn't the first part of the check redundant, given the outer if condition?

Reply:
No because this is about the reconnect being undef which means we have a forgotten player, but you're right, there might be a better if/else logic to do that, I'll think about it

Reply:
Oh, right, had the logic swapped in my head. Nevermind :-)